src/BepInEx/Translation/en/Text/GuildHouse/Game Table/Other text.txt

```diff
@@ -0,0 +1,35 @@
+---Other---
+
+み=E
+んなのゲームテーブル=veryone's Game Table
+
+ニゲーム=nigame
```

Comment:
Wouldn't this show up as Enigame because of the み=E rule?

Reply:
The "E" is for the "veryone's Game Table" :D
The "Mi" for the "nigame" is should be in some other translation file as far as I know.
I tested, it showed up fine as "Minigame" for me.

Reply:
Oh right, the "Mi" used in that case would be Katakana instead, right.
I guess it would make sense to remove this "first letter" separation and to combine the two labels into one, I might do that sometime. Would need updates of the translations though.

Reply:
I guess this split is to be able to have the first letter bigger, but I wouldn't mind joining them if that only sacrifices that.
This split is causing weird texts in many places, and sometimes that starting letter cannot be matched to the rest of the text as all.
Eg. for "Event", the starting letter is an "I", so the full text had to be "IEvent"..